### DISCUSSION TITLE
How to download characters from server with SSC enabled?

### DISCUSSION THREAD

**Discussion by zeroarst** [Pryaxis/TShock#1974] [Q&A]
I host the server with my laptop, and I use another PC to connect to it and play. I was initially thinking to host it and let my friends play but apparently they are are not interested in SSC feature...
So I decide to stop the server and just play on my own...

but the problem is... how can I download the character from my tshock server?? I can access the server...

**Comment by hakusaro** [comment]
When the server is in SSC mode, the client won't write out data. You could open the character database and then copy the inventory or give yourself items one by one. That's probably the easiest thing unless someone knows of a converter.
